connectors/es/sink.py

```diff
@@ -576,6 +654,7 @@ async def async_bulk(
         sync_rules_enabled=False,
         content_extraction_enabled=True,
         options=None,
+        job_type=JobType.FULL,
```

Comment:
Would it be more secure if we leave `job_type` empty and then validate so anyone who calls `async_bulk` explicitly knows what they are doing?